README.md

```diff
@@ -32,6 +32,18 @@ Community
 
 Stay updated by following our [Twitter](https://twitter.com/Sylius) and [Facebook](https://www.facebook.com/SyliusEcommerce/).
 
+Official plugins
+----------------
+
+This is the list of oficially supported Sylius plugins:
+
+ - [Shop API Plugin](https://github.com/Sylius/SyliusShopApiPlugin) - 
+   provides customer-level API for the shop functionalities
+ - [Admin Order Creation Plugin](https://github.com/Sylius/AdminOrderCreationPlugin) - 
+   allows to create orders directly from the administrator panel
+ - [Customer Reorder Plugin](https://github.com/Sylius/CustomerReorderPlugin) - 
+   allows customer to copy items from the already placed order to the current cart
```

Comment:
I would mention that `AdminOrderCreationPlugin` and `CustomerReorderPlugin` are at the beginning of development, to not mislead anybody (could be simple `[WIP]` tag)

Reply:
Shouldn't we mention that in plugins' readme first? :)

Reply:
Agreed, however, I think it's worth to mention it here as well - to communicate clearly that it's not finished/released yet.

Reply:
I'm afraid that we'd need to mark all of those `[WIP]` as Shop APi is in beta and the other two have `0.x` releases only

Reply:
Shop API is in beta, then it only needs some polishing and improvements and can be released as stable. AdminOrderCreation and CustomerReorder provide about half of the desired features 😄 it's a pretty clear difference for me
But I don't insist ☮️